lib/src/logic.dart

```diff
@@ -424,41 +424,47 @@ class Logic {
   }
 
   /// Accesses the [index]th bit of this signal.
+  /// Negative/Positive index values are allowed. (The negative indexing starts from where the array ends)
```

Comment:
Add an additional blank line `///` between the main description (first line) and more detailed information.